### PR TITLE
Update Okta docs with the latest UI values

### PR DESCRIPTION
**Pull request by robnewman**
# Description
Add latest info from the updated Okta UI

# TODO
- [X] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

**Comment by robnewman** [comment]
@akarve - Would love to get these updated Okta docs added. Please review

**Comment by robnewman** [comment]
@akarve <boop> Should be a quick review

**Review by drernie — DISMISSED** [review]

File: docs/technical-reference.md [hunk 1]
```diff
@@ -287,24 +287,40 @@ and hybrid flows, and check the box to issue ID tokens
...
-1. Name the app `Quilt` or something similar
-1. Configure the app as shown below
-1. Add `<QuiltWebHost>` to `Login redirect URIs` and
-`Initiate login URI`
-1. Copy the `Client ID` to a safe place
-1. Go to API > Authorization servers
-1. You should see a default URI that looks something like the following:
+Note: You will need Okta administrator privileges to add a new Application.
+
+1. Go to **Okta > Admin > Applications > Applications**
```
Comment (akarve) [inline docs/technical-reference.md:292]: For style consistency with the rest of docs, no bold here.

File: docs/technical-reference.md [hunk 2]
```diff
@@ -287,24 +287,40 @@ and hybrid flows, and check the box to issue ID tokens
...
+
+2. Click `Create App Integration`. A new modal window opens.
+3. Assign `Sign-in method` radio button to `OIDC - OpenID Connect`.
+4. Assign `Application type` radio button to `Web Application`.
+
+![](./imgs/okta-add-application-modal.png)
+
+5. Click the `Next` button.
+6. Rename the default `App integration name` to `Quilt` or something distinctive for your organization to identify it.
+7. Optionally add the Quilt logo for user recognition.
```
[the remark this answers is not in the document]
Comment (akarve) [reply docs/technical-reference.md:304]: Yes! 7. Shouldn't be optional. Link directly to this image (and host it in our repo).
quilt-okta

File: docs/technical-reference.md [hunk 3]
```diff
@@ -287,24 +287,40 @@ and hybrid flows, and check the box to issue ID tokens
...
+
+1. Go to **Okta > Admin > Applications > Applications**
+
+![](./imgs/okta-add-application.png)
+
+2. Click `Create App Integration`. A new modal window opens.
+3. Assign `Sign-in method` radio button to `OIDC - OpenID Connect`.
+4. Assign `Application type` radio button to `Web Application`.
+
+![](./imgs/okta-add-application-modal.png)
```
Comment (akarve) [inline docs/technical-reference.md:300]: @robnewman be sure to "crush" all images with ImageOptim or PNG crush or similar so repo doesn't bloat.

Comment (robnewman) [reply docs/technical-reference.md:300]: All net-new images added are optimized

File: docs/technical-reference.md (hunk 1 above)
Comment (robnewman) [reply docs/technical-reference.md]: Bold removed

File: docs/technical-reference.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (robnewman) [reply docs/technical-reference.md]: Removed 'optional' text, and linked to image referenced

**Review by drernie — APPROVED** [review]

**Pull request merged**